README.md

```diff
@@ -19,12 +19,28 @@ individual rules.
 [2]: https://github.com/markdownlint/markdownlint/blob/master/docs/creating_styles.md
 [3]: https://github.com/markdownlint/markdownlint/blob/master/docs/RULES.md
 
+### `directory`
+
+**Optional** The path to run `mdl` against when linting markdown files.
+Defaults to current directory `.` if a directory is not specified.
+
 ## Example usage
 
+### with style-file
```

Comment:
Can you capitalize the first word in this heading and below?

Reply:
Certainly.
Done.